src/main/java/reposense/util/FileUtil.java

```diff
@@ -40,6 +40,8 @@
     private static final ByteBuffer buffer = ByteBuffer.allocate(1 << 11); // 2KB
 
     private static final String MESSAGE_INVALID_FILE_PATH = "\"%s\" is an invalid file path. Skipping this directory.";
+    private static final String MESSAGE_FAIL_TO_ZIP_FILES =
+            "Exception occurred while attempting to ZIP the contents of the report files.";
```

Comment:
My idea: `Exception occurred while attempting to ZIP the report files.` is enough.